microcosm_daemon/daemon.py

```diff
@@ -113,7 +113,9 @@ def run(self):
 
         if args.processes < 1:
             parser.error("--processes must be positive")
-        elif args.processes == 1:
+        elif args.processes == 1 and args.heartbeat_threshold_seconds < 0:
```

Comment:
Isn't `heartbeat_threshold_seconds` optional? Logic should check if provided and non-negative.

Reply:
There's a default value specified for it in the arg parser.